dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -69,6 +72,9 @@ def self.get_courses(state_code, row_hash)
     when 'ID'
       # A column per CS course with a value of 'Y' if the course is offered.
       ['02204',	'03208', '10157'].select {|course| row_hash[course] == 'Y'}
+    when 'SC'
+      # One source per row
+      [row_hash['coursecode']]
```

Comment:
My thought was if we ought to change this from:
`[row_hash['coursecode']]`
to:
`[UNSPECIFIED_COURSE]`

With the `UNSPECIFIED_COURSE` constant defined to be "unspecified"

Reply:
Oh, I had wondered that too.  Will make the change.

Reply:
agree with drew